ts/sagas/startup/watchMessageLoadSaga.ts

```diff
@@ -0,0 +1,63 @@
+import { TypeofApiCall } from "italia-ts-commons/lib/requests";
+import { buffers, Channel, channel } from "redux-saga";
+import { call, fork, take } from "redux-saga/effects";
+import { ActionType, getType } from "typesafe-actions";
+
+import { GetUserMessageT } from "../../../definitions/backend/requestTypes";
+import { loadMessage as loadMessageAction } from "../../store/actions/messages";
+import { loadMessage } from "../messages/messages";
+
+// Here we set the number of handlers we want to create.
+// This means that we will have at most a number of concurrent
+// fetchs (of the message detail) equal to the number of the handlers.
+const NUMBER_OF_HANDLERS = 5;
+
+/**
+ * This generator listen for loadMessage.request actions and forward them
+ * to a poll of created handlers.
+ *
+ * @param getMessage API call to fetch the message detail
+ */
+export function* watchMessageLoadSaga(
+  getMessage: TypeofApiCall<GetUserMessageT>
+) {
+  // Create the channel used for the comunication with the handlers.
+  const requestsChannel: Channel<
+    ActionType<typeof loadMessageAction.request>
+  > = yield call(channel, buffers.expanding());
```

Comment:
from [the docs](https://github.com/redux-saga/redux-saga/blob/master/docs/api/README.md#buffers) it looks like we need to specify an initial size, if there's a default, add it to the comment above

Reply:
Yes there is a default size of 10

https://github.com/redux-saga/redux-saga/blob/2aecdb310db8760ddd2c2513bf1cce7174c289d9/packages/core/src/internal/buffers.js#L85
https://github.com/redux-saga/redux-saga/blob/2aecdb310db8760ddd2c2513bf1cce7174c289d9/packages/core/src/internal/buffers.js#L12

Reply:
pls add this information to the comment above